src/main/java/com/ibm/nfvodriver/service/impl/LcmOpOccPollingService.java

```diff
@@ -0,0 +1,74 @@
+package com.ibm.nfvodriver.service.impl;
+
+import static com.ibm.nfvodriver.config.NFVODriverConstants.COMPLETED_OPERATIONAL_STATES;
+
+import java.io.IOException;
+import java.util.Collections;
+import javax.annotation.PreDestroy;
+
+import com.ibm.nfvodriver.service.ExternalMessagingService;
+import org.etsi.sol005.lifecyclemanagement.LcmOperationStateType;
+import org.etsi.sol005.lifecyclemanagement.VnfLcmOpOcc;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+
+import com.ibm.nfvodriver.driver.NSLifecycleManagementDriver;
+import com.ibm.nfvodriver.model.LcmOpOccPollingRequest;
+import com.ibm.nfvodriver.model.alm.ExecutionAsyncResponse;
+import com.ibm.nfvodriver.model.alm.ExecutionStatus;
+import com.ibm.nfvodriver.model.alm.FailureDetails;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class LcmOpOccPollingService {
```

Comment:
Any reason for adding the new class LcmOpOccPollingService? Are you using it anywhere in the service class?

Reply:
in kafka configure class , earlier  we have commented the some code now we enabled that code